data/json/snippets/nightmares.json

```diff
@@ -53,7 +53,73 @@
+      "You awaken from a nightmare about being stoned to death, the crowd full of wide-eyed maniacs, some holding pipes and crowbars.",
+      "You dream of losing your mind, forgetting all that you know, and becoming one of the many.",
+      "You have a nightmare about being in a car crash.  As you crawl free of the wreckage, broken and bleeding, the horde descends upon you.",
+      "You dream of a small child tearing out your throat with their blackened teeth.",
+      "You have a nightmare in which you walk through a river of blood, followed by the dead you tried to leave behind.",
+      "You dream of slime, jiggling, wobbling, and oozing over your entire body.  As you wipe at it, the flesh of your hand smears away.",
+      "You dream of standing on the Eiffel Tower, watching Paris burn.  You’re too far above to make out who is still alive and who isn’t, though you hear the screams.",
+      "You dream of a beautiful garden full of strange plants that defend you against the undead.  When they’ve finished, their barbed tendrils turn on you, paralyzing and restraining you until you wake screaming.",
+      "You dream that you’re a time traveler.  You go back to tell your past self about the horrors to come, but after hours of pleading, you just won’t listen.",
+      "You have a nightmare about fighting for days against a warrior clad in bone armor.  Teeth clatter gently, woven into their hair.  When finally you are defeated, the warrior removes their mask, but you wake before seeing their face.",
+      "You dream you are a slug, and all is well until a child stands over you with a salt shaker.",
+      "You dream you’re walking in the woods.  There’s no one around and your phone is dead.  That’s when you spot him.",
+      "You wake from a nightmare of being slowly devoured, starting with your toes.",
+      "You wake from a nightmare of being torn to pieces, lying in the filth of the road and waiting for the end.",
+      "A seemingly endless nightmare plagues you.  Years pass, and you are always struggling alone in an unforgiving world, dying in solitude of old age.",
+      "You dream that you are the last person on Earth.  You’re surrounded by \"people\" but you know they aren’t human anymore.",
+      "You wake from a nightmare in which you endlessly stacked boxes in a factory, overseen by an inhuman figure clothed in shadow.",
+      "You dream of a behemoth, towering over you and covered in bony tissue, its face twisted with rage.  You run for as long as you can, but when its massive fist comes down, you wake.",
+      "You dream of a beautiful and regal unicorn.  You feed it gems as it nuzzles your hand.  When you’ve run out of stones, it kicks you to death.",
+      "You wake from a nightmare in which it had all started over again, 911 ringing busy, the fires, and death in every street.",
+      "You dream your boss is scolding you for allowing a zombie through security.",
```

Comment:
```suggestion
      "You dream that your boss is scolding you for allowing a zombie through security.",
```

Reply:
I prefer the original for this one, I don't know if it matters